commit bf38dfdfef4a22cbaaaa75d5080446a9b0eac156 (refs/changes/64/4264/3)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2018-08-01T21:23:31+00:00 (1 year, 2 months ago)
    
    MANATEE-401 postgresql should not reuse old WAL files
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/deps/postgresql96 b/deps/postgresql96
index 563881d..5f50a8f 160000
--- a/deps/postgresql96
+++ b/deps/postgresql96
@@ -1 +1 @@
-Subproject commit 563881d704954a95174be84330af3e218e30143b
+Subproject commit 5f50a8f2b09c0e35d72d878c3d498a221aeeb4b8
diff --git a/etc/9.6/postgresql.manta.coal.conf b/etc/9.6/postgresql.manta.coal.conf
index dc6a3da..8fd2fc8 100644
--- a/etc/9.6/postgresql.manta.coal.conf
+++ b/etc/9.6/postgresql.manta.coal.conf
@@ -652,3 +652,4 @@ default_text_search_config = 'pg_catalog.english'
 #------------------------------------------------------------------------------
 
 # Add settings for extensions here
+wal_recycle = off
diff --git a/etc/9.6/postgresql.manta.lab.conf b/etc/9.6/postgresql.manta.lab.conf
index d745fe8..af89a11 100644
--- a/etc/9.6/postgresql.manta.lab.conf
+++ b/etc/9.6/postgresql.manta.lab.conf
@@ -652,3 +652,4 @@ default_text_search_config = 'pg_catalog.english'
 #------------------------------------------------------------------------------
 
 # Add settings for extensions here
+wal_recycle = off
diff --git a/etc/9.6/postgresql.manta.production.conf b/etc/9.6/postgresql.manta.production.conf
index 72f5d84..fac0e2e 100644
--- a/etc/9.6/postgresql.manta.production.conf
+++ b/etc/9.6/postgresql.manta.production.conf
@@ -653,3 +653,4 @@ default_text_search_config = 'pg_catalog.english'
 #------------------------------------------------------------------------------
 
 # Add settings for extensions here
+wal_recycle = off
